### ISSUE TITLE
the accuracy i get is low

### ISSUE THREAD

**Issue by Winnie202** [MCG-NJU/VideoMAE#13]
i train the model on UCF101, pretrain （100 epochs,finetune（100 epochs）, the accuracy is 67-68,but your accuracy is above 90， Is it related to parameter settings? can you tell me the parameter settings

**Comment by yztongzhan** [comment]
Hi @Winnie202! Thanks for your interest in our VideoMAE.  Our VideoMAE is pretrained for **3200** epochs on 
UCF101. Please check our paper for more details.

**Comment by Winnie202** [comment]
Thanks a lot for your answer. Could you please tell me where I can download pretrained checkpoint on UCF101?
